lib/Db/Helper.php

```diff
@@ -21,6 +21,14 @@
 
 class Helper
 {
+    /**
+     *
+     * @param array{dbColumName: string, dbColumnValue: mixed} $data The actual data to be inserted or updated into the database table.
```

Comment:
Are you sure this is right? It means an array with the two keys `['dbColumName' => '...', 'dbColumnValue' => ...]` should be passed. I don't see anywhere that the keys `dbColumName` or `dbColumnValue` are used.

You probably mean:
```suggestion
     * @param array<string, mixed> $data The actual data to be inserted or updated into the database table.
```